libs/lib3/project.json

```diff
@@ -65,14 +65,14 @@
       "configurations": {
         "production": {
           "commands": [
-            "rm -rf dist/libs/lib3",
+            "node -e \"require('fs').rmSync('dist/libs/lib3', {recursive: true, force: true})\"",
```

Comment:
Let's instead create a simple `tools/scripts/remove-dir.js` script that receives the directory to remove. The usage here would be simpler and less error-prone.

Reply:
I agree with you on that. I just pushed, take a look. Make adjustments if needed.

It seems that Nx should be able to do this removal of the 'dist' folder. I noticed some Nx plugins have a `deleteOutputPath` switch, but unfortunately '@nrwl/workspace' doesn't.